Tests/SWBBuildSystemTests/CustomTaskBuildOperationTests.swift

```diff
@@ -35,6 +35,11 @@ fileprivate struct CustomTaskBuildOperationTests: CoreBasedTests {
                 environment = [:]
             }
 
+            var toolName = "tool"
+            if let ext = destination.imageFormat(core).executableExtension.nilIfEmpty {
```

Comment:
nit: there's an `executableName(basename:)` which automates this logic, can you use that?